bin/pkg/validate_lock_dir.ml

```diff
@@ -0,0 +1,79 @@
+open! Import
+open Pkg_common
```

Comment:
I find this a bit confusing to follow (since there's a bunch of names that are unqualified in the code) but that's just me.